src/components/NewMessage/NewMessage.vue

```diff
@@ -82,6 +82,7 @@
 					dir="auto"
 					@keydown.esc="handleInputEsc"
 					@keydown.ctrl.up="handleEditLastMessage"
+					@keydown.meta.up="handleEditLastMessage"
```

Comment:
meta is a 'Windows' key also. maybe replace it with `useHotKey()`?

Reply:
> meta is a 'Windows' key also

It is not a problem, it is ignored on Windows and, afaik, all (most?) Linux DEs (Checked on GNOME, KDE, xfce, Cinnamon)

> maybe replace it with `useHotKey()`?

I'd avoid having a global listener here